src/Gui/NotificationBox.cpp

```diff
@@ -218,9 +231,13 @@ bool NotificationLabel::eventFilter(QObject *o, QEvent *e)
             auto remaining = expireTimer.remainingTime();
             auto lapsed = total - remaining;
             // ... or if the click is inside the notification, hide it no matter if the minimum onscreen time has lapsed or not
-            if( lapsed > minShowTime || this->underMouse()) {
+            auto insideclick = this->underMouse();
+            if( lapsed > minShowTime || insideclick) {
                 hideNotification();
-                return false;
+
+                if(insideclick) {
+                    return true;
```

Comment:
`return insideclick;` ?

Reply:
Ok. 

I cannot change it right now, as I am working on another PR.

Reply:
No problem. Very minor and as said, I let you manage merging on your own. ;)

Reply:
I have pushed the change.